server/src/fat/java/com/ibm/ws/lars/rest/ApiTest.java

```diff
@@ -1146,11 +1209,16 @@ public void test500Mapping() throws InvalidJsonAssetException, ParseException, I
     @Test
     public void testAssetReviews() throws InvalidJsonAssetException, IOException {
         Asset a = addLittleAsset("weather", "hot", "ground", "flat", "description", "lovely");
-        List<?> ars = repository.getAssetReviews(a.get_id());
+        List<Map<String, Object>> ars = repository.getAssetReviews(a.get_id());
         // Story 165844, for now assetreviews just returns an empty JSON array
-        assertEquals(
-                     "Should be an empty List",
-                     0, ars.size());
+        assertEquals("Should be an empty List", 0, ars.size());
+        RepositoryContext userRepo = RepositoryContext.toUserContext(repository);
+        try {
```

Comment:
yes, but it didn't seem worth adding it for just the one instance. When the review api is implemented more fully and there are more tests, it would be worth a writing a 'get in error' method. I didn't think it was worth it for the moment